src/roles/remove-iptable-rules/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
We also have a role that allows xmpp connections by removing the iptables rule that drops them, I think that can be used instead of a new role

Reply:
Done